### DISCUSSION TITLE
Changing my profile country

### DISCUSSION THREAD

**Discussion by michael-rowe** [streetcomplete/StreetComplete#4275] [Q&A] [answered]
Hi there. Apologies if I'm asking in the wrong place but not sure where else to look.

I moved to the UK from South Africa a few months ago and recently noticed that, even though my OSM profile information has been updated to reflect my UK details, Streetcomplete is still giving me stats (e.g. my country rank) in SA.

Where can I change this?

**Comment by Helium314** [comment, marked as answer]
As far as I understand the code, it always shows the country where you solved most of your quests. So the "main" country should switch after a while (assuming you use the app of course).

**Reply by michael-rowe** [reply]
Thank you.
